src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -73,26 +73,100 @@
  * &lt;module name="MethodLength"/&gt;
  * </pre>
  * <p>
- * To configure the check so that it accepts methods with at most 60 lines:
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *   public MyClass() {  // constructor (line 1)
+ *        /&#42; line 2
+ *            ...
+ *           line 150 &#42;/
+ *   } // line 151, violation, as it is over 150
+ *
+ *   public void firstExample() { // line 1
+ *
+ *       // line 3
+ *       System.out.println("line 4");
+ *       /&#42; line 5
+ *          line 6 &#42;/
+ *   } // line 7, OK, as it is less than 150
+ *
+ *   public void secondExample() { // line 1
+ *       // line 2
+ *       System.out.println("line 3");
+ *
+ *       /&#42; line 5
+ *           ...
+ *          line 150 &#42;/
+ *   } // line 151, violation, as it is over 150
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it accepts methods with at most 4 lines:
  * </p>
  * <pre>
  * &lt;module name="MethodLength"&gt;
  *   &lt;property name="tokens" value="METHOD_DEF"/&gt;
- *   &lt;property name="max" value="60"/&gt;
+ *   &lt;property name="max" value="4"/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check so that it accepts methods with at most 60 lines,
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyTest {
+ *   public MyTest()  {  // constructor (line 1)
+ *       // line 2
+ *       // line 3
+ *       // line 4
```

Comment:
Please use some java code here 
You can keep comments as training comments.
`int i; // line 2

When limit is small, we can make code some sort of real.